zitadel/local/README.md

```diff
@@ -0,0 +1,166 @@
+
+Click on the `+` button to open the `Add a Manager` pop-up window, then click on the name of newly created service user 
+as item under the `Loginname` and tick the `Iam Owner` as role. Finally, click on the `Add` button to finish this step.
+![add-user-as-iam-owner](compose/statics/add-user-as-iam-owner.png)
+
+### Run Terraform Scripts to initialize the configuration.
+
+Enter the directory 'zitadel/terraform' under the project:
+
+```ssh
+cd ../terraform
+ ```
+Read the document 'README.md' and check preparation before running terraform scripts. What we need to pay attention to
+completing as below:
+
+* Copy the json file we have already downloaded to the path and rename it to 'local-token.json'.
+* Use the 'local.ftvars' to connect to the local service instance of Zitadel as below:
+
+```ssh
+terraform init
+terraform plan -var-file=environments/local.tfvars -var-file=environments/environment.tfvars
+terraform apply -auto-approve -var-file=environments/local.tfvars -var-file=environments/environment.tfvars
+```
+
+After successfully executing the Terraform script, the Terraform outputs is as below:
+![terraform-output-config-items](compose/statics/terraform-outputs-config.png)
+
+The default IAM admin user login the console of Zitadel to view added configurations:
+![view-configs-on-console](compose/statics/view-configs-on-console.png)
+
+### Add Admin User of Xpanse
+
+To facilitate local development of the xpanse project, it is necessary to create an admin user with all roles of
+'xpanse'. Do following action to crate the admin user on the console:
+
+Click on the name `eclipse-xpanse` in the list of `ORGANIZATIONS` to open the organization management page for the newly 
+created organization named `eclipse-xpanse`.
+![open-org-manage-page.png](compose/statics/open-org-manage-page.png)
+
+Click on the menu `Users` and the `+ New` button on the console in sequence to open page `Create a New User` as below.
+Fill all required fields and then click the button `Create` to create the new user.
+![create-user-for-xpanse](compose/statics/create-user-for-xpanse.png)
+
+> Note: The first login requires a password change, please remember the new password.
+
+Follow the steps below to grant all roles of 'xpanse' for the new admin user:
+1. Select item `Authorizations` under the user and then click on the button `+ New N` to start the grant;
+![grant-user-roles-start](compose/statics/grant-user-roles-start.png)
+2. Select the project `eclipse-xpanse`and then click on the button `Continue`;
+![img.png](compose/statics/grant-select-project.png)
+3. Select all roles to the grant and then clock on the button `Save`;
+![img.png](compose/statics/grant-select-all-roles.png)
+After completing the grant above, user has roles as below:
+![img.png](compose/statics/view-user-has-all-roles.png)
+
+## Run Applications of Xpanse with Local Service of Zitadel
+
+### Config and run the API service of Xpanse.
```

Comment:
same here. I would not recommend temporarily updating any files that are checked-in into repo.
Even here, we must only show how to provide these values as environment variables and start the server. 

1. for fixed values such as url, we can permanently checkin the zitadel-local properties file. 
2. start server with zitadel  and zitadel-local profile and env variables for Zitadel properties. 

I think we can also remove 'zitadel-testbed' from default profiles. Then the developer must explicitly select what he wants to do.

Reply:
Updated it by https://github.com/eclipse-xpanse/xpanse/pull/786